src/orange/OrangeData.hh

```diff
@@ -156,13 +156,12 @@ struct RectArrayRecord
     // Daughter data [index by LocalVolumeId]
     ItemMap<LocalVolumeId, DaughterId> daughters;
 
+    //! Cursory check for validity
     explicit CELER_FUNCTION operator bool() const
     {
-        return daughters.size() > 0
-               && daughters.size()
-                      == (grid[to_int(Axis::x)].size() - 1)
-                             * (grid[to_int(Axis::y)].size() - 1)
-                             * (grid[to_int(Axis::z)].size() - 1);
+        return !daughters.empty() && !grid[to_int(Axis::x)].empty()
```

Comment:
This no longer checks that the number of daughters and the grid size are consistent?